test/Data/String/QQ.hs

```diff
@@ -0,0 +1,19 @@
+-- | This module is based off the QQ implementation from string-qq
+-- (https://github.com/audreyt/string-qq).
```

Comment:
What was the license for string-qq, and does this use conform to it?

Reply:
The original package is listed as public domain.